sr_grasp_stability/script/tf_computation.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python

```

Comment:
I asked you to submit a PR with only a package skeleton. These files should not be in the skeleton.
The idea of the skeleton was that James could start working in parallel more quickly using version control.